packages/esm-patient-registration-app/src/patient-registration/field/address/address-field.component.tsx

```diff
@@ -23,42 +24,48 @@ export const AddressField: React.FC = () => {
   const { addressTemplate } = useContext(ResourcesContext);
   const [addressFields, setAddressFields] = useState([]);
   const { t } = useTranslation();
-  const addressTemplateXml = addressTemplate.results[0].value;
+  const addressTemplateXml = addressTemplate?.results[0].value;
 
   useEffect(() => {
-    const templateXmlDoc = parseString(addressTemplateXml);
-    const nameMappings = getTagAsDocument('nameMappings', templateXmlDoc);
-    const elementDefaults = getTagAsDocument('elementdefaults', templateXmlDoc);
-    const properties = nameMappings.getElementsByTagName('property');
-    const propertiesObj = Array.prototype.map.call(properties, (property: Element) => {
-      const name = property.getAttribute('name');
-      /*
+    if (addressTemplateXml) {
+      const templateXmlDoc = parseString(addressTemplateXml);
+      const nameMappings = getTagAsDocument('nameMappings', templateXmlDoc);
+      const elementDefaults = getTagAsDocument('elementdefaults', templateXmlDoc);
+      const properties = nameMappings.getElementsByTagName('property');
+      const propertiesObj = Array.prototype.map.call(properties, (property: Element) => {
+        const name = property.getAttribute('name');
+        /*
```

Comment:
I know that this is not your code, but I'm wondering whether the following comment means anything to anyone. Maybe we can just straight up remove it? 

![grafik](https://user-images.githubusercontent.com/30902964/166915728-53568537-287a-438c-935e-6d5b3d20d9e3.png)

Reply:
Yes, I'll remove this comment block too.

Reply:
@vasharma05 Would you mind re-adding that comment? Comments like this are how we tell i18next that we still need translations for these keys even though they aren't included here (because we literally call `t(name, ...)` to load the label text).

Reply:
Sure @ibacher!